Specs/Scene/Cesium3DTilesetSpec.js

```diff
@@ -4866,6 +4869,241 @@ describe(
+          return originalLoadJson(tilesetUrl).then(function (tilesetJson) {
+            var extension =
+              tilesetJson.root.extensions["3DTILES_multiple_contents"];
+            extension.content = [
+              {
+                uri: "nonexistent1.b3dm",
+              },
+              {
+                uri: "nonexistent2.b3dm",
+              },
+              {
+                uri: "nonexistent3.b3dm",
+              },
+            ];
+
+            return tilesetJson;
+          });
+        });
+
+        var uris = [];
+
+        viewNothing();
+        return Cesium3DTilesTester.loadTileset(scene, multipleContentsUrl).then(
+          function (tileset) {
+            tileset.tileFailed.addEventListener(function (event) {
+              console.log(event);
+              uris.push(event.url);
+            });
+
+            viewAllTiles();
+            return Cesium3DTilesTester.waitForTilesLoaded(scene, tileset).then(
+              function () {
+                expect(uris.length).toBe(3);
+                uris.sort();
+                var expected = [
+                  "nonexistent1.b3dm",
+                  "nonexistent2.b3dm",
+                  "nonexistent3.b3dm",
+                ];
+
+                for (var i = 0; i < expected.length; i++) {
+                  expect(uris[i].endsWith(expected[i])).toBe(true);
+                }
+              }
+            );
+          }
+        );
+      });
+
+      it("raises tileFailed for external tileset inside multiple contents", function () {
+        var originalLoadJson = Cesium3DTileset.loadJson;
+        spyOn(Cesium3DTileset, "loadJson").and.callFake(function (tilesetUrl) {
+          return originalLoadJson(tilesetUrl).then(function (tilesetJson) {
+            var extension =
+              tilesetJson.root.extensions["3DTILES_multiple_contents"];
+            extension.content = [
+              {
+                uri: "external.json",
+              },
+            ];
+
+            return tilesetJson;
+          });
+        });
+
+        spyOn(Resource.prototype, "fetchArrayBuffer").and.callFake(function () {
+          var externalTileset = {
+            asset: {
+              version: "1.0",
+            },
+            geometricError: 100.0,
+          };
+          var buffer = Cesium3DTilesTester.generateJsonBuffer(externalTileset);
+          return when.resolve(buffer);
+        });
+
+        viewNothing();
+        return Cesium3DTilesTester.loadTileset(scene, multipleContentsUrl).then(
+          function (tileset) {
+            tileset.tileFailed.addEventListener(function (event) {
+              expect(event.url.endsWith("external.json")).toBe(true);
+              expect(event.message).toEqual(
+                "External tilesets are disallowed inside the 3DTILES_multiple_contents extension"
+              );
+            });
+
+            viewAllTiles();
+            scene.renderForSpecs();
```

Comment:
Also check that the `tileFailed` event is triggered. Currently this test would always pass I think.